doc/whats_new.rst

```diff
@@ -35,6 +35,11 @@ API
 - :func:`~hnn_core.Dipole.write` and :func:`~hnn_core.Dipole.read_dipoles`
   now support hdf5 format for read/write Dipole object, by
   `Rajat Partani`_ in :gh:`648`
+  
+- Add ability to optimize parameters associated with evoked drives and plot 
+  convergence. User can constrain parameter ranges and specify solver. 
+  Document with an example and develop tests, by `Carolina Fernandez Pujol`_ 
```

Comment:
The last sentence is unnecessary since examples + developing tests should be implicit with any significant change to the API. Short and concise is better here.

Reply:
Done